keyboards/handwired/dactyl_manuform/5x6/keymaps/swedish/keymap.c

```diff
@@ -0,0 +1,56 @@
+/* A Swedish XMonad layout for the Dactyl Manuform 5x6 Keyboard */ 
+
+#include QMK_KEYBOARD_H
+#include "keymap_swedish.h"
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
```

Comment:
You may want to set this up as an enum, as this may be easier to maintain long term.

Reply:
Good idea, fixed.